atlp-rwanda/e-comm-team-techsmith-fn
# #185218921 Seller should create or add product

## What does this PR do?

A seller should be able to create/add items

## Description of Task to be completed?

- The seller should be able to add/create items for sale
- Seller can add different 4 picture of product created/added
- form validation using react hooks is implemented


## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout <ft-seller-should-create-and-add-items-185218921>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm run dev`

- Check the application's documentation on Storybook: `npm run storybook`



## What are the relevant pivotal tracker/Trello stories?
- [185218921](https://www.pivotaltracker.com/story/show/185218921)

## Screenshots (if appropriate)
![Screen Shot 2023-06-14 at 10 07 24](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/77435896/8d967ddb-8899-48f3-9504-ae1a6cdbfe58)
